packages/neuron-wallet/src/subjects/addressesUsedSubject.ts

```diff
@@ -0,0 +1,16 @@
+import { ReplaySubject } from 'rxjs'
+
+// subscribe this Subject to monitor which addresses are used
+export class AddressesUsedSubject {
```

Comment:
The name is a bit confusing, `AddressesUsedSubject` is not a subject, `AddresesUsedSubject.subject` and `AddressesUsedSubject.getSubject()` are subjects.

Reply:
rename to `AddressesUsedEvent` ? or any advice